lib/crypto/crypto_keys_subr.c

```diff
@@ -32,7 +32,7 @@
  */
 
 static int import_BN(BIGNUM **, const uint8_t **, size_t *);
-static int export_BN(BIGNUM *, uint8_t **, size_t *, uint32_t *);
+static int export_BN(const BIGNUM *, uint8_t **, size_t *, uint32_t *);
```

Comment:
and yes, SSLeasy didn't use `const` here, at least as of 0.9.1b.  But is that a concern for us?